src/types/project.d.ts

```
@@ -1,31 +1,29 @@
-
-import 'express';
-export { }
+import "express";
+export {};
```

Biome (1.9.4)

[error] This empty export is useless because there's another export or import.

This import makes useless the empty export.

Safe fix: Remove this useless empty export.


(lint/complexity/noUselessEmptyExport)